projects/dsp-ui/src/lib/viewer/views/list-view/list-view.component.ts

```diff
@@ -78,15 +120,16 @@ export class ListViewComponent implements OnChanges {
         this.view = view;
     }
 
-    emitSelectedResource(id: string) {
-        // get selected resource index from list to highlight it
-        for (let idx = 0; idx < this.resources.resources.length; idx++) {
-            if (this.resources.resources[idx].id === id) {
-                this.selectedResourceIdx = idx;
-                break;
-            }
+    // If 'withMultipleSelection' is true, multiple resources are selected for comparision
+    // If 'withMultipleSelection' is false, single resource is selected for viewing
+    emitSelectedResources(resInfo: FilteredResouces) {
+        this.selectedResourceIdx = resInfo.resListIndex;
+
+        if (resInfo.selectionType === "multiple") {
+            this.multipleResourcesSelected.emit(resInfo);
+        } else {
+            this.singleResourceSelected.emit(resInfo.resIds[0]);
```

Comment:
the old/deprecated `resourceSelected` emitter is missing. Please keep `this.resourceSelected.emit(id);` Otherwise it doesn't work anymore in some apps

Reply:
done